test/integration/stamp_editor_spec.js

```diff
@@ -100,7 +116,11 @@ describe("Stamp Editor", () => {
           await input.uploadFile(
             `${path.join(__dirname, "../images/firefox_logo.svg")}`
           );
-          await page.waitForSelector(`${getEditorSelector(1)} .altText`);
+          // With Chrome on Windows, it seems that the SVG is not loaded
+          // immediately, so we wait a bit to avoid an issue when serializing
+          // the editor.
+          await page.waitForTimeout(500);
```

Comment:
This seems unfortunate, especially given the effort both you and Tim have put into removing "arbitrary" timeouts lately, and most likely it'll only be a question of time (pun intended) before this causes new intermittent failures.

Reply:
Yes the exception we get is coming from:
https://chromium.googlesource.com/chromium/src/third_party/+/refs/heads/main/blink/renderer/core/offscreencanvas/offscreen_canvas.cc#230
I suppose that's because the image is a svg and it isn't not completely drawn when we transfer to a bitmap.
I think the real issue is that `transferToImageBitmap` should be async (or better `drawImage` should be async).
That said maybe I've an idea (an awful but it could work, let me try).